.github/workflows/sync.yml

```diff
@@ -4,6 +4,8 @@ name: sync test
 
 on:
   merge_group:
+  push:
+    branches: [main, scroll]
```

Comment:
Currently, we don't have a merge queue so I believe the first event `merge_group` will never trigger and we will not do any pre-merge validation of sync. This is probably fine for now the post-merge `push` trigger should be sufficient but I will add an issue to introduce a merge queue as the repo begins to stabilize and we are moving into production readiness.